client/src/components/ElementEditor/Element.js

```diff
@@ -165,12 +167,12 @@ const Element = (props) => {
 
   // Publish action
   useEffect(() => {
-    if (formHasRendered && doPublishElement) {
+    if (doPublishElement) {
```

Comment:
Seems like this bit should not be getting removed? Checking the formHasRendered is presumably still required for regular inline editable blocks?